modules/test-utils/src/index.js

```diff
@@ -1,9 +1,9 @@
-export {makeSpy} from './make-spy';
+export {makeSpy} from './utils/make-spy';
```

Comment:
`make-spy` is not a util, it is a major exported function/symbol from test-utils.